### DISCUSSION TITLE
Transcription fails on speaker change

### DISCUSSION THREAD

**Discussion by J-Korn** [openai/whisper#2358] [Q&A]
I am running a self-hosted Whisper server to transcribe Greek news videos using the Large-v3 model. However, I'm encountering an issue where the transcription sometimes fails after a speaker change. In some cases, it resumes after a sentence or two, but other times it completely fails to understand the second speaker.
Both speakers have excellent audio quality, and neither has a thick accent or is difficult to understand.
Is there anything I can do to improve this?

**Comment by ryanheise** [comment]
There are various options that can help. Are you able to share the audio file?
